DickensJuma/Enumerables
# Rspec feature to Develop

### Test Enumerables using **Rspec**

- Add specs for all Enumerable methods

- Wrap specs for each method in describe block

- Define a variable for the test cases, use let

- Use one 'it' block for each expected behaviour

- Test positive and negative scenarios
